worker_types/gwci-linux/userdata

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+mkdir -p /home/ubuntu/generic-worker
+cd /home/ubuntu/generic-worker
+apt update
+apt upgrade -y
+apt install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
+apt update
+apt-cache policy docker-ce | grep -qF download.docker.com
+apt install -y docker-ce
+sleep 5
+systemctl status docker | grep "Started Docker Application Container Engine"
+usermod -aG docker ubuntu
+id -nG | grep docker
+curl -L https://github.com/taskcluster/generic-worker/releases/download/v15.1.2/generic-worker-simple-linux-amd64 > generic-worker
```

Comment:
let's pull version numbers out into environment variables so they're easier to see